tests/test_flask_sqlalchemy_session.py

```diff
@@ -28,7 +28,7 @@ def flask_app():
 
 @pytest.fixture
 def unitialized_session(sqlite_engine):
-    """Return a request_scoped_session that has not been used for a flask app"""
+    """Return a request_scoped_session that has not been used for flask app"""
```

Comment:
If you can please remove this change, as it results in a grammatically incorrect sentence, I will merge the PR. Thanks!

Reply:
Actually I just did that